proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11846,6 +11846,50 @@ function longAllAreRepeated( test )
   var expected = false;
   test.identical( got, expected );
 
+  /* with evaluator, equalizer */
+
+  test.case = 'empty array with evaluator';
+  var evaluator = ( e ) => e;
+  var src = [];
+  var got = _.longAllAreRepeated( src, evaluator );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'empty array with equalizer';
+  var equalizer = ( e1, e2 ) => e1.val === e2;
+  var src = [];
```

Comment:
Long type is a complex type. So, add same test cases for unrolls. See [all Long types](https://github.com/dmvict/wTools/blob/master/doc/ukr/concept/TypeIndexed.md)